frontend/src/components/Accordion.vue

```diff
@@ -41,7 +41,7 @@ export default {
         contentHeight: function () {
             if (this.isOpen) {
                 const content = this.$refs.content
-                return content.scrollHeight + 'px'
+                return (2 * content.scrollHeight) + 'px'
```

Comment:
Why is doubling the scrollHeight needed here? `contentHeight` is setting the maxHeight, right, which is used for the accordion animation, or is there something else it's handling I'm missing?

Reply:
It's doubled due to the new error `<detail>` sections introduced - it was my dirty quick fix. When you expand the `<summary>`, the panel size doesn't update with it, if we set the max height to double, it gives us plenty of wiggle room to expand multiple error messages, without fear of exceeding. Alongside this, i also double the animation time, so that then behaves/animates at the speed we would have wanted prior to introducing the doubling.